Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -40,22 +49,42 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
     // Make sure it is initialized before changing the startup mode.
     _ = MSTransmissionTargets.shared
 
+    // Storage size section.
+    let storageMaxSize = UserDefaults.standard.integer(forKey: kMSStorageMaxSizeKey)
+    self.storageMaxSizeField.text = "\(storageMaxSize / 1024)"
+    self.storageMaxSizeField.addTarget(self, action: #selector(storageMaxSizeUpdated(_:)), for: .editingChanged)
+    if let supportDirectory = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first {
+      let dbFile = supportDirectory.appendingPathComponent("com.microsoft.appcenter").appendingPathComponent("Logs.sqlite")
+      func getFileSize(_ file: URL) -> Int {
+        return (try? file.resourceValues(forKeys:[.fileSizeKey]))?.fileSize ?? 0
```

Comment:
I think the default max size printed shouldn't be zero, the default max size from the SDK would be ideal but we won't access it from all the Sasquatch app  so I would say we should print a `-` or anything but a number. That way we know it hasn't been set.